.gitignore

```diff
@@ -0,0 +1,28 @@
+
+# Azure Functions artifacts
+bin
+obj
+appsettings.json
+local.settings.json
+
+# Azurite artifacts
+__blobstorage__
+__queuestorage__
+__azurite_db*__.json
```

Comment:
Can we can have a .gitignore specific to azure in the AzureFunctions folder? It'll work only if these files are generated in that folder.